<launch>
	<!-- <include file="$(find openni2_launch)/launch/openni2.launch"/> -->
	<!-- <include file="$(find astra_launch)/launch/astra.launch" /> -->
	<node name="depth_scan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" output="screen" args="image:=/camera/aligned_depth_to_color/image_raw">
		<remap from="depth_image" to="/camera/aligned_depth_to_color/image_raw" />
		<remap from="camera_info" to="/camera/aligned_depth_to_color/camera_info" />
		<remap from="scan" to="/depth_scan_points" />
		<param name="scan_height" value="400" />
		<param name="output_frame_id" value="camera_depth_frame" />
		<param name="range_min" type="double" value="00" />
		<param name="range_max" type="double" value="10.0" />
		<param name="ythresh_min" type="double" value="-0.03" />
		<param name="ythresh_max" type="double" value="-0.02" />
		<param name="center_dx" type="double" value="-0.025" />
		<param name="center_dy" type="double" value="-0.07" />
		<param name="center_dz" type="double" value="-0.0" />
	</node>
	
	<!-- <node name="laser_tf"         pkg="depthimage_to_laserscan"  type="laser_tf" output="screen" >
        <remap from="merg_res" to="/scan1"/>
    </node> -->
</launch>
